src/components/Chat/index.js

```diff
@@ -49,8 +50,71 @@ class Chat extends Component {
   renderMessages = () =>
     this.props.messages.map(msg => <Message type={msg.author} content={msg.content} />);
 
+  renderTyping = () =>
```

Comment:
would be nice to have this in a `lib` of some kind eventually since it's shared between operator and client... someday...

Reply:
Yeah, even if its obfusticated in a shared library that they both import would be nice, but right now it would just be this one svg aha. Anyways, agreed!

Reply:
👍